workflows/sarek.nf

```diff
@@ -344,8 +344,8 @@ workflow SAREK {
     rt_file                = PREPARE_GENOME.out.rt_file
 
     // Gather index for mapping given the chosen aligner
-    ch_map_index = params.aligner == "bwa-mem" ? bwa :
-        params.aligner == "bwa-mem2" ? bwamem2 :
+    ch_map_index = params.aligner == "bwamem" ? bwa :
+        params.aligner == "bwamem2" ? bwamem2 :
```

Comment:
might have to add `bowtie2` and `snap` here too

Reply:
I'll do it in another PR, need to add the index as well, and docs, and probably test this.